test/library/standard/Random/RandomStreamInterface/choiceTest.chpl

```diff
@@ -113,19 +113,21 @@ proc testArray(stream, arr: [], size:?sizeType=none, replace=true, prob:?probTyp
   if isDomainType(sizeType) then m = size.size;
   else if isIntegralType(sizeType) then m = size;
 
-
-  var actualRatios = if isNothingType(sizeType) then counts / trials:real
-                     else counts / (trials*m): real;
-
-
+  var actualRatios = new map(int, real);
+  for (k,v) in actualRatios.items() {
```

Comment:
The `items()` iterator yields by `const ref`, so we can't assign directly to `v` here. I think the reason it is `const` is so that you can't modify the keys. If I change it to yield by `ref` you can write it with your shortcut.